<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiberAssistantDialogBase</class>
 <widget class="QDialog" name="FiberAssistantDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fiber Assistant</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { background-color: #F0F0F0; color: #333333; } </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="top_bar_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="logo_label">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap>:/icons/FA_baner.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="settings_layout">
        <item>
         <widget class="QPushButton" name="refresh_button">
          <property name="toolTip">
           <string>Odśwież</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/options_icons/options_icon_refresh.png</normaloff>:/icons/options_icons/options_icon_refresh.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="export_button">
          <property name="toolTip">
           <string>Eksportuj treść okna do pliku .txt</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/options_icons/options_icon_export_txt.png</normaloff>:/icons/options_icons/options_icon_export_txt.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copy_button">
          <property name="toolTip">
           <string>Kopiuj do schowka</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/options_icons/options_icon_copy.png</normaloff>:/icons/options_icons/options_icon_copy.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clean_button">
          <property name="toolTip">
           <string>Wyczyść zawartość okna</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/options_icons/options_icon_clean.png</normaloff>:/icons/options_icons/options_icon_clean.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settings_button">
          <property name="toolTip">
           <string>Ustawienia</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/options_icons/options_icon_settings.png</normaloff>:/icons/options_icons/options_icon_settings.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="help_button">
          <property name="toolTip">
           <string>Pomoc</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/options_icons/options_icon_help.png</normaloff>:/icons/options_icons/options_icon_help.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="main_menu_widget">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="right_panel_layout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     
     <item>
      <widget class="QStackedWidget" name="function_stacked_widget">
       <widget class="QWidget" name="page_wyszukiwarka"/>
       <widget class="QWidget" name="page_statystyka"/>
       <widget class="QWidget" name="page_walidator"/>
       <widget class="QWidget" name="page_przeliczanie_dlugosci"/>
       <widget class="QWidget" name="page_dane_podstawowe_projektu"/>
       <widget class="QWidget" name="page_zarzadzanie_kablami"/>
       <widget class="QWidget" name="page_zarzadzanie_PA"/>
       <widget class="QWidget" name="page_zarzadzanie_PE"/>
       <widget class="QWidget" name="page_karta_krosowan"/>
       <widget class="QWidget" name="page_stycznosc_wierzcholkow"/>
       <widget class="QWidget" name="page_wykorzystanie_infrastruktury"/>
       <widget class="QWidget" name="page_elementy_niewybudowane"/>
       <widget class="QWidget" name="page_raport_miesieczny_qgis"/>
       <widget class="QWidget" name="page_raport_polroczny_qgis"/>
       <widget class="QWidget" name="page_funkcjonalnosci_dla_tok"/>
       <widget class="QWidget" name="page_uzupelnianie_struktury_projektu"/>
       <widget class="QWidget" name="page_czyszczenie"/>
       <widget class="QWidget" name="page_funkcje_w_fazie_testow"/>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="bottom_bar_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="left_panel" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="show_logs_button">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Wyświetl konsolę z pełnymi logami</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_10px">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status_label">
        <property name="text">
         <string>Gotowy</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_stretch">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="run_button">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Uruchom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>